login-utils/lslogins.c

```diff
@@ -1639,6 +1708,11 @@ int main(int argc, char *argv[])
 		case OPT_LASTLOG:
 			path_lastlog = optarg;
 			break;
+#ifdef HAVE_LASTLOG2
+		case OPT_LASTLOG2:
+		        path_lastlog2 = optarg;
```

Comment:
This could write directly to `ctl->lastlog2_path`.

Reply:
Difficult, because you need to initialize ctl->lastlog2_path somehow with the default path.

Reply:
Could be done below instead of `ctl->lastlog2_path = path_lastlog2;`.